free-3.md

```diff
@@ -57,12 +139,22 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Most of this project would be spend creating the architecture that translate the
+markup document to  human readable form since the langugae will only have very
+basic grammars and words and mostly rely on custom extensibility by the user.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+While this would be a neat project, it is not overly ambitious since its scope
+is small and does not require a complicated syntax. The grammar for the project
+would be fairly easy to define since it would be intentionally limited.
```

Comment:
I think having a lot of tooling (i.e. a good GUI) might make this helpful. Like @jonathanFinnell mentioned, typing out these descriptive names every time would be a pain. Alternatively, maybe having an IDE with intelligent autocomplete would do the job.